### PR TITLE
test: date_histogram with time zone on date_nanos field

### PR DESCRIPTION
**Pull request by salvatore-campagna**
Include `date_histogram` test on `date_nanos` field.
The test runs fine without any fix.

Issue: #39107

### CODE REVIEW

**Comment by salvatore-campagna** [comment]
@elasticmachine update branch

**Comment by not-napoleon** [comment]
This test can just land in the existing `360_date_histogram.yml` file

File: rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/480_date_histogram.yml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+          - { "date": "2015-01-01T12:10:30.123456789Z" }
+          - { "index": {} }
+          - { "date": 1420070400 }
+          - { "index": {} }
+
+---
+"date_histogram on date_nanos without timezone fixed interval":
+  - skip:
+      version: " - 8.1.99"
+      reason: bug fixed in 8.2.0
```
Comment (not-napoleon) [inline rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/480_date_histogram.yml:30]: The linked issue lists https://github.com/elastic/elasticsearch/pull/52016 as resolving this, which landed in 7.6.1.  Let's confirm it works there or at least 7.7.0 and update the skip version appropriately.  As noted in your other PR, I'm in favor of backporting these tests to 7.17.

**Review by not-napoleon — APPROVED** [review]
LGTM

**Comment by salvatore-campagna** [comment]
@elasticmachine update branch

**Comment by salvatore-campagna** [comment]
@elasticmachine update branch

**Pull request merged**